bin/inx-chronicle/config.template.toml

```diff
@@ -12,4 +12,5 @@ port = 9092
 allow_origins = ["0.0.0.0"]
 
 [syncer]
-max_simultaneous_requests = 10
+earliest_milestone = 0
```

Comment:
Now that we chatted about it, I like the idea, but wouldn't use a fixed milestone index, but rather a `delay` or `max_milestones_to_request` as we have in Bee and Hornet.